assets/aircraft/e175.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Embraer 175",
+  "icao": "E175",

```

Comment:
I see you are adding the Embraer ERJ-175 in addition to the existing ERJ-170. The ICAO identifiers for those two aircraft are actually the same, because they are (essentially) the same model. Technically speaking, I think the 175 may be 10 feet longer (or less), and hold another two or so rows of passengers. Thus, ICAO treats them as the same model, as does the FAA:

![image](https://cloud.githubusercontent.com/assets/5103735/14430885/db5ff3bc-ffc8-11e5-99db-bc5c28061021.png)
![image](https://cloud.githubusercontent.com/assets/5103735/14430985/47433724-ffc9-11e5-857f-dac495e8289d.png)

The best course then, I think, would be just to leave both in the `e170.json` file only, under the icao identifer of `E170`.